applications/GeoMechanicsApplication/custom_conditions/README.md

```diff
@@ -0,0 +1,141 @@
+
+- $\epsilon$ = effective emissivity  $\mathrm{[-]}$
+- $\sigma$ = Stefan Boltzmann's constant  $\mathrm{[5.67 \cdot 10^{-8}  W/m^{2 \circ}C^4]}$
+- $T_{at}$ = atmospheric temperature $\mathrm{[^{\circ}C]}$
+
+And the emitted long wave radiation,
+
+$$ R^{l,em} = \sigma \left( T_{ss} +273.15 \right)^4 $$
+
+where $T_{ss}$ $\mathrm{[^{\circ}C]}$ is land surface temperature.
+
+## Surface Heat Storage
+The surface heat storage term $Q_s$ $\mathrm{[W/m^2]}$ is calculated by,
+
+$$ Q_s =  a_1 R_n(t+\Delta t) + a_2 \frac{R_n(t+\Delta t) - R_n(t)}{\Delta t} + a_3 $$
+
+where $a_1$ $\mathrm{[-]}$, $a_2$ $\mathrm{[s]}$ and $a_3$ $\mathrm{[W/m^2]}$ are user defined variables.
+
+
+## Evaporation Flux
+The evaporation flux $E$ $\mathrm{[m/s]}$ is calculated by,
+
+$$ E = \frac{L_v}{\lambda \rho_w} $$
+
+where
+
+- $L_v$ = latent heat flux
+- $\lambda$ = latent heat of vaporization $\mathrm{[2.45 \space MJ/kg]}$
+- $\rho_w$  = density of water $\mathrm{[kg/m^3]}$
+
+The latent heat flux is:
+
+$$ L_v  = \frac{e^\prime_{at} \left(R_n + Q_f - Q_s \right) + C_a \rho_a \left(e_{at}^s - e_{at}^a\right) / r_a}{e^\prime_{at} + \gamma (1+r_{s} / r_a)} $$
+
+where
+
+- $r_a$ = atomospheric resistance $\mathrm{[s/m]}$
+- $r_s$ = surface resistance constant $\mathrm{[30 \space s/m]}$ 
+- $\gamma$ = psychometric constant $\mathrm{[0.67 \space hPa/ ^{\circ}C]}$
+- $Q_f$ = build enviromental radiation $\mathrm{[W/m^2]}$ (user defined parameter)
+- $C_a$ = specific heat of moist air $\mathrm{[J/kg ^{\circ}C]}$
+- $\rho_a$ = air density $\mathrm{[kg/m^3]}$ 
+- $e^s$ = saturated vapour pressure $\mathrm{[hPa]}$ 
+- $e^\prime$ = slope of the saturation vapor curve $\mathrm{[hPa/ ^{\circ}C]}$
+- $e^a$ = actual vapour pressure $\mathrm{[hPa]}$
+
+$$ e^s = 6.11 \exp \left( \frac{17.27 \space T}{T+237.3} \right) $$
+
+$$ e^\prime=  \frac{4098 \space e^s}{\left(T + 237.3\right)^2} $$   
+
+$$ e^a = \frac{\mathrm{RH}}{100} e^s $$
+
+$\mathrm{RH}$ is the relative humidity which is a user defined parameter.
+
+The atomospheric resistance,
+
+$$ r_a = \frac{1}{0.007+0.0056 u} $$
+
+where $u$ $\mathrm{[m/s]}$ is the wind speed which is a user defined parameter. $t$ $\mathrm{[s]}$ and $\Delta t$ $\mathrm{[s]}$ are time and time step, respectively.
+
+## Roughness Layer Temperature
+The uenergy balance equation for the roughness layer reads as,
+
+$$ h_{rl} \frac{\partial T_{rl}}{\partial t} = \frac{T_{ss}-T_{rl}}{r_g} + u f_h a_d^2 \left(T_{at}-T_{rl}\right)  $$
+
+After discritization,
+
+$$ h_{rl} \frac{T_{rl}(t+\Delta t) - T_{rl}(t)}{\Delta t} = \frac{T_{ss}(t)-T_{rl}(t+\Delta t)}{r_g} + u f_h a_d^2 \left[T_{at}(t+\Delta t)-T_{rl}(t+\Delta t) \right] $$
+
+Then the temperature in the roughness layer $T_{rl}$ reads as,
+
+$$ T_{rl}(t+\Delta t) = \frac{r_g \space h_{rl} \space T_{rl}^0 + \Delta t \space T_{ss}(t) + r_g \space \Delta t \space u \space f_h \space d_d^2 \space T_{at}(t+\Delta t)}{r_g \space h_{rl} + \Delta t + r_g \space \Delta t \space u \space f_h \space a_d^2} $$
```

Comment:
Perhaps, a  notation from other source and T_{rl}^0 should be T_{rl}(t), correct?